src/tseda/datastore.py

```diff
@@ -115,6 +114,10 @@ def tooltip(self):
             ),
         )
 
+    def sample_indices(self):  # TODO: make sure this updates
```

Comment:
is this TODO finished or should it still be fixed in this/another PR?

Reply:
It is indeeed fixed!